vault/request_handling.go

```diff
@@ -271,6 +271,16 @@ func (c *Core) handleLoginRequest(req *logical.Request) (*logical.Response, *log
 		return nil, nil, ErrInternalError
 	}
 
+	// The token store uses authentication even when creating a new token,
+	// so it's handled in handleRequest. It should not be reached here.
+	if strings.HasPrefix(req.Path, "auth/token/") {
+		c.logger.Printf(
```

Comment:
`gq` is what I use personally. I probably have `set tw=<something>` too but I'm just used to using `gq`.